src/aasthaJJun20/PrimeNumbers.java

```diff
@@ -0,0 +1,51 @@
+package aasthaJJun20;
+
+public class PrimeNumbers {
+	//a) Given number is prime or not. 
+		boolean isPrimeNumber(int num){
+			for(int index = 2; index <= Math.sqrt(num); index++){
+				if(num%index == 0){
+					return false;
+				}
+			}
+			return true;
+		}
+		
+		// b) Print total count of prime numbers between 2->100.
+		int findTotalPrimeInRange(int start, int end){
+			int count = 0;
+			for(int index = start; index <= end; index++){
+				if(isPrimeNumber(index))
+					count++;
+			}
+			return count;
+		}
+		
+		//c) Find sum of all prime numbers between 2->100
+		int findSumOfPrimesInRange(int start, int end){
+			int sum = 0;
+			for(int index = start; index <= end; index++){
+				if(isPrimeNumber(index))
+					sum += index;
+			}
+			return sum;
+		}
+		
+		//d) Find avg of prime numbers between 2->100
+		double findAvgOfPrimesInRange(int start, int end){
+			return Double.valueOf(findSumOfPrimesInRange(start,end))/Double.valueOf(findTotalPrimeInRange(start,end));
```

Comment:
You just need to take two instance variable one for Sum and count.
And implement both logic in isPrimeNumber() method then no need to write an extra two methods. 
ANd printing statement itself you can calculate sum/count = avg.

Reply:
Hi Krishna,
Sorry I misinterpreted the requirement, I thought we have to write a method for each mentioned point.
As per your comment, I have removed 3 methods for - sum, count, average and also removed the if-else statements from main method so that the whole logic stays in display methods. Please review further.
Thanks.
![PrimeNumbers-output](https://user-images.githubusercontent.com/66835733/85949722-020e6a00-b976-11ea-9875-0dfc763a9ca2.png)